CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+1.0.5
+=====
+*   (improvement) Extract `filterOptionalFields` to `FilterFieldsHelper`.
```

Comment:
```suggestion

*   (improvement) Extract `filterOptionalFields` to `FilterFieldsHelper`.
```

Please keep the same formatting as in the rest of the file (= one blank line after the headline)